src/libasr/codegen/llvm_utils.h

```diff
@@ -170,28 +170,32 @@ namespace LFortran {
                                 llvm::Module& module);
 
             llvm::Value* read_item(llvm::Value* list, llvm::Value* pos,
-                                   bool get_pointer=false, bool check_index_bound=true);
+                                   bool enable_bounds_checking,
+                                   llvm::Module& module, bool get_pointer=false);
 
             llvm::Value* len(llvm::Value* list);
 
-            void check_index_within_bounds(llvm::Value* list, llvm::Value* pos);
+            void check_index_within_bounds(llvm::Value* list, llvm::Value* pos,
+                                           llvm::Module& module);
 
             void write_item(llvm::Value* list, llvm::Value* pos,
                             llvm::Value* item, ASR::ttype_t* asr_type,
-                            llvm::Module& module, bool check_index_bound=true);
+                            bool enable_bounds_checking, llvm::Module& module);
```

Comment:
I think, there will be no need for bounds_checking in write_item.
@czgdp1807 What do you think?